Mathlib/CategoryTheory/Bicategory/Product.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2025 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne
+-/
+module
+
+public import Mathlib.CategoryTheory.Bicategory.Functor.StrictPseudofunctor
+
+/-!
+# Cartesian products of bicategories
+
+We define the bicategory instance on `B × C` when `B` and `C` are bicategories.
+
+We define:
+* `sectL B c` : the strictly unitary pseudofunctor `B ⥤ B × C` given by `X ↦ ⟨X, c⟩`
+* `sectR b C` : the strictly unitary pseudofunctor `C ⥤ B × C` given by `Y ↦ ⟨b, Y⟩`
+* `fst`       : the strict pseudofunctor `⟨X, Y⟩ ↦ X`
+* `snd`       : the strict pseudofunctor `⟨X, Y⟩ ↦ Y`
+* `swap`      : the strict pseudofunctor `B × C ⥤ C × B` given by `⟨X, Y⟩ ↦ ⟨Y, X⟩`
+
+-/
+
+@[expose] public section
+
+namespace CategoryTheory.Bicategory
+
+open Prod
+
+universe w₁ w₂ v₁ v₂ u₁ u₂
+
```

Comment:
```suggestion
universe w₁ w₂ v₁ v₂ u₁ u₂

attribute [local simp] CategoryTheory.prod_Hom

```

Reply:
This is a good point, but since the issue is that very basic simp lemmas don't work, shouldn't the change be made global? Otherwise, I think others in the future will run into the same issues as me.